### ISSUE TITLE
TokenStore - Define init/1 required by behaviour GenServer

### ISSUE THREAD

**Issue by nitinstp23** [peburrows/goth#37]
Getting this warning message on compilation - 

```ex
warning: function init/1 required by behaviour GenServer is not implemented (in module Goth.TokenStore).

We will inject a default implementation for now:

    def init(args) do
      {:ok, args}
    end

But you want to define your own implementation that converts the arguments given to GenServer.start_link/3 to the server state
```

**Comment by peburrows** [comment]
Thanks, I've merged #36 and published to hex as `0.8.1`!

**Comment by nitinstp23** [comment]
@peburrows  Cool 👍
